packages/transaction-preview/src/views/Maximized.svelte

```diff
@@ -42,7 +42,14 @@
 
   const cleanBalance = (dirtyBalance: string): string => {
     const gweiToEther = ethers.utils.formatEther(dirtyBalance)
-    const roundTo4Decimal = parseFloat(gweiToEther).toFixed(4)
+    const roundTo4Decimal = parseFloat(gweiToEther).toFixed(6)
```

Comment:
Don't put the number of decimals IMO as then we get issues like this where it was changed from `4` to `6` and now you have to update the variable names every time precision changes 
```suggestion
    const roundGwei = parseFloat(gweiToEther).toFixed(6)
```
@Adamj1232

Reply:
Artifact of days of ol....Good catch thank you!

Reply:
haha figured! lol apologies for the tone I just re-read that lmao
added some fun flair 😝